index.html

```diff
@@ -5,23 +5,85 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>To-Dos App</title>
 </head>
-  <body>
-      <h1>To-Dos</h1>
-      <form>
-        <input type="text" placeholder="Enter a task">
-        <input type="submit" value="+">
-      </form>
-      <form>
-        <input type="checkbox" id="checkbox1" value="dummy1" checked>
-        <label for="checkbox1"><s>Completed Task</s></label><br>
-        <input type="checkbox" id="checkbox2" value="dummy2">
-        <label for="checkbox2">Incomplete Task</label>
-        <button type="submit">Submit Response</button>
-      </form>
-      <menu>
-        <li><button>All Tasks</button></li>
-        <li><button>Complete Tasks</button></li>
-        <li><button>Incomplete Tasks</button></li>
-      </menu>
-  </body>
+<body>
+  <header>
+    <h1>To-Dos
+    </h1>
+  </header>
+  <nav>
+    <button>
+      <svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" fill="currentColor" class="bi bi-file-earmark-check-fill" viewBox="0 0 14 14">
+      <path d="M9.293 0H4a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h8a2 2 0 0 0 2-2V4.707A1 1 0 0 0 13.707 4L10 .293A1 1 0 0 0 9.293 0zM9.5 3.5v-2l3 3h-2a1 1 0 0 1-1-1zm1.354 4.354-3 3a.5.5 0 0 1-.708 0l-1.5-1.5a.5.5 0 1 1 .708-.708L7.5 9.793l2.646-2.647a.5.5 0 0 1 .708.708z"/>
```

Comment:
Since `<path>` is a nested inside `<svg>`, it should have another level of indentation.